ariadne_codegen/client_generators/result_types.py

```diff
@@ -163,6 +174,16 @@ def generate(self) -> ast.Module:
             )
         return module
 
+    def include_model_rebuild(self, class_def: ast.ClassDef) -> List[ast.Expr]:
+        model_rebuild_calls = []
+        for node in ast.walk(class_def):
+            if isinstance(node, ast.Name) and '"' in node.id:
```

Comment:
You can return `True` as soon as you find that model should have a `model_rebuild` call added for it.

Likewise, function could return `False` otherwise.